README.md

```diff
@@ -72,6 +72,17 @@ const developerOAuth = new DeveloperOAuth({
 }),
 ```
 
+##### Trust Proxy
+If your app is behind proxy, please set trust proxy in express.
+Without doing so, `X-Forward-Proto` may not able to pass along and affecting the protocol used in redirection.
```

Comment:
Without doing so, it might affect the protocol, ip and host of the `Request` object passed in when express needs to read real values from `X-Forward-` headers